chapter03/dags/03_with_end_date.py

```diff
@@ -31,7 +31,10 @@ def _calculate_stats(input_path, output_path):
     calculate_stats = PythonOperator(
         task_id="calculate_stats",
         python_callable=_calculate_stats,
-        op_kwargs={"input_path": "/tmp/data/events.json", "output_path": "/tmp/data/stats.csv",},
+        op_kwargs={
+            "input_path": "/tmp/data/events.json",
```

Comment:
/tmp is not needed. When using the new docker-compose /data is a volume in all containers.

Reply:
Thats completely right, we already mounted the data directory as well, I will correct that